addons/mail/static/src/models/message/message.js

```diff
@@ -331,6 +331,16 @@ registerModel({
         _computeCanStarBeToggled() {
             return !this.messaging.isCurrentUserGuest && !this.isTemporary && !this.isTransient;
         },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeDatetime() {
```

Comment:
reorder alphabetically

Reply:
It's alphabetical. `datetime` < `dateDay`.